man/org.freedesktop.systemd1.xml

```diff
@@ -620,6 +611,27 @@ node /org/freedesktop/systemd1 {
       appended to <filename>/sys/fs/cgroup/systemd</filename> easily. This value will be set to the empty
       string for the host instance, and some other string for container instances</para>
     </refsect2>
+
+    <refsect2>
+      <title>Security</title>
+
+      <para>Read access is generally granted to all clients. All operations are allowed for clients with the
+      <constant>CAP_SYS_ADMIN</constant> capability. Additionally, for unprileged clients, some operations
```

Comment:
The CAP_SYS_ADMIN stuff is a lie, unfortunately. This worked on kdbus only. On AF_UNIX dbus-daemon cannot safely determine the client's caps, hence it uses an euid check instead. It's one of the things I'd really like to see added to AF_UNIX one day though, hence I don't think we should remove specification of the cap in code. but we probably shouldn't document it as doing anything.

Reply:
Duh, I didn't check that what the code does.